go/launcher/client/client-otlp.go

```diff
@@ -0,0 +1,110 @@
+//
+// example code to test lightstep/opentelemetry-exporter-go
```

Comment:
not sure what this comment block means. maybe it should be removed altogether

Reply:
Yeah, me neither. Was scratching my head on that one!

Reply:
Cleaned up the comment to make it more meaningful